Sources/LanguageServerProtocol/SupportTypes/VersionedTextDocumentIdentifier.swift

```diff
@@ -42,7 +42,7 @@ public struct OptionalVersionedTextDocumentIdentifier: Hashable, Codable, Sendab
   /// identifier is sent from the server to the client and the file is not
   /// open in the editor (the server has not received an open notification
   /// before) the server can send `null` to indicate that the version is
-  /// known and the content on disk is the master (as specified with document
+  /// known and the content on disk is the master (as specified with document # ignore-unacceptable-language
```

Comment:
I realize this is the LSP docs, but... seems fine to just update it to `primary`. Especially to avoid the `#ignore-unacceptable-language` showing up in the docs 😅